app/assets/scripts/vendor/map.js

```diff
@@ -140,29 +127,31 @@
                   markersToShow.push(markers[i]);
               }
           }
-          var table = $("<table border=1></table>");
+          var table = $("<table border=0></table>");
           $.each(markersToShow, function(i, markerToShow) {
             tableCell = document.createElement('td');
             $(tableCell).attr({
               'style': 'overflow:hidden;width:100%;height:100%;position:absolute'
             });
             tableCellBoldTitle = document.createElement('strong');
             tableCellNewline = document.createElement('br');
-            $(tableCellBoldTitle).html(markerToShow.getTitle());
-            $(tableCell).append(tableCellBoldTitle, 
-                                tableCellNewline, 
-                                markerToShow.getLabel());
             tableCellInnerDiv = document.createElement('div');
             $(tableCellInnerDiv).attr({
               'style': 'width:50px;height:50px; text-align:right; float: right;'
             });
-            tableCellImg = document.createElement('img');
+            tableCellImg = document.createElement('img');            
             $(tableCellImg).attr({
-              'src': markerToShow.getIcon().url,
-              'style': 'width:50px;height:50px;'
-            });
-            $(tableCellInnerDiv).append(tableCellImg);
+              'src': 'http://www.google.com/intl/en_us/mapfiles/ms/micons/red-dot.png',
```

Comment:
please download this img and have this code extract it from local storage so we're not dependent on google keeping the URL the same